certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Ticket",
```

Comment:
@developmentLL we have an existing connector with the same name "[Tesseron ASM Ticket](https://learn.microsoft.com/en-us/connectors/tesseronasmticket/)", can you please confirm if this is an update to the existing connector or is it a new connector? if it is new, then please change the title.